genrl/deep/agents/ppo1/ppo_temp.py

```diff
@@ -0,0 +1,166 @@
+from typing import Any, Dict, Optional, Tuple, Union
+
+import gym
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as opt
+from torch.autograd import Variable
+
+from ....environments import VecEnv
+from ...common import (
+    BaseOnPolicyAgent,
+    RolloutBuffer,
+    get_env_properties,
+    get_model,
+    load_params,
+    save_params,
+    set_seeds,
+)
+
+
+class PPO1(BaseOnPolicyAgent):
+    def __init__(
+        self,
+        network_type: str,
+        env: Union[gym.Env, VecEnv],
+        timesteps_per_actorbatch: int = 256,
+        gamma: float = 0.99,
+        clip_param: float = 0.2,
+        actor_batch_size: int = 64,
```

Comment:
Do we even need this parameter? I don't see it being used anywhere.

Reply:
Actor_batch_size?

Reply:
Yes

Reply:
Yeah probably not after batch size

Reply:
It wasn't use anywhere even before, I'll remove this.